internal/slicer/report_test.go

```diff
@@ -200,7 +207,26 @@ var reportTests = []struct {
 	add: []sliceAndEntry{
 		{entry: fsutil.Entry{Path: "/file"}, slice: oneSlice},
 	},
-	err: `cannot add path "/file" outside of root "/base"`,
+	err: `cannot add path: "/file" outside of root "/base"`,
+}, {
+	summary: "Add mutated regular file",
```

Comment:
Can you add another entry here or another test where one entry is not mutated? just to see that the mutation is restricted to the path

Reply:
Added.